examples/official-storybook/manager-head.html

```diff
@@ -2,4 +2,10 @@
   #manager-head-not-loaded {
     display: none;
   }
+  
+  @media (hover: none) {
+    div > div > div > ul {
```

Comment:
Can we make this more specific, with a class or an id? This seems like a potentially dangerous selector

Reply:
I agree